README.md

```diff
@@ -0,0 +1,88 @@
+# SoundBubble
+
+###### Group Members: [Martin Palanca](https://github.com/martipal), [Sherry Zhang](https://github.com/veronicz), [Yancey Yang](https://github.com/Talos6)
+
+SoundBubble is a platform that leverages Spotify's public API to build logs of a user's recently played tracks and makes this data visible to other users. Users can create and join groups to refine which users' song logs appear on their feed, as well as rate songs using an 'thumbs up' and 'thumbs down' scale.
+
+## Table of Contents
+
+- [Project Description](#Project-Description)
+- [Project Task Requirements](#Project-Task-Requirements)
+  - [Minimal Requirement](#Minimal-Requirement)
+  - [Standard Requirements](#Standard-Requirements)
+  - [Stretch Requirements](#Stretch-Requirements)
+- [Breakdown of Minimal Requirements](#Breakdown-of-Minimal-Requirements)
+  - [Requirement 1: Retrieve a user’s recently played song log](#Retrieve-a-users-recently-played-song-log)
+  - [Requirement 2: Create a user interface](#Create-a-user-interface)
+- [Prototypes](#Prototypes)
+- [MongoDB Schema](#MongoDB-Schema)
+
+## Project Description
+
+This project is intended for users who are subscribed to the music platform Spotify. It will allow users, and other users within their “Group”, to view each other’s song logs and interact with the logs by “Up Voting” or “Down Voting” a particular log. The type of data we will be storing are Songs along with User Information. This data will be able to be shared among users to access within the group. Additional functionality that we may add/remove based on time constraints is the ability to recommend a played song to another user, and adding commenting functionality to a particular song log for users to provide their opinions (e.g. “This song is awesome!”, “This song is lame”).
+
+## Project Task Requirements
+
+### Minimal Requirement
+
+1. Retrieve a user’s recently played song log
+2. Create a user interface for interacting with Spotify song logs
+3. The ability to “Up Vote” and “Down Vote” a particular song log
+4. The ability to add users to a private group
+5. User log in capabilities
+
+### Standard Requirements
+
+1. Be able to clear a user’s posted song logs
```

Comment:
do we have this one? or put this one into the stretch

Reply:
yeah I don't think we have this

Reply:
I think we interpreted this as hide/show song logs? I can reword the requirement to make that clear.